_partials/_cloud-installation.md

```diff
@@ -1,21 +1,73 @@
-## Create a Timescale Cloud account
+## Create a $CLOUD_LONG account
 
-You create a $CLOUD_LONG account to manage your services and data in a centralized and efficient manner. From $CONSOLE you can create and delete services, run queries, manage access and billing, integrate other services, contact support, and more.
+You create a $CLOUD_LONG account to manage your $SERVICE_SHORT and data in a centralized and efficient manner in $CONSOLE. From $CONSOLE_SHORT, you can create and delete $SERVICE_SHORTs, run queries, manage access and billing, integrate other services, contact support, and more.
+
+<Tabs label="Create a Timescale Cloud account">
+
+<Tab title="Standalone">
```

Comment:
```suggestion
<Tab title="$CONSOLE">
```
That is, if the variable works.

Reply:
No, the variable does not work. My concern is that in 1. both cases you create an account in Timescale Console, you just start from AWS in one case. 1. I use "standalone" not only in the tab.

Reply:
Standalone makes me think of self-hosted TimescaleDB, rather than the Cloud solution. I guess you could say Timescale billing and AWS billing or something like that.